library/alloc/src/vec.rs

```diff
@@ -2281,7 +2281,7 @@ where
 
         // use try-fold since
         // - it vectorizes better for some iterator adapters
-        // - unlike most internal iteration methods methods it only takes a &mut self
+        // - unlike most internal iteration methods, it only takes a &mut self
         // - it lets us thread the write pointer through its innards and get it back in the end
```

Comment:
Is this even valid? Looks weird to me.

Reply:
Not sure what in particular you're referring to.

Reply:
> thread the write pointer through its innards

Reply:
to thread [is a verb](https://en.wiktionary.org/wiki/thread#Verb), and innards is figurative expression for internal components.